tests/regression.rs

```diff
@@ -644,3 +644,17 @@ fn test_hough_line_detection() {
 
     compare_to_truth_image(&lines_image, "hough_lines.png");
 }
+
+#[test]
+fn test_bilateral_filter() {
+    fn bilat_filt(image: &GrayImage) -> GrayImage {
```

Comment:
Nit: can we just call this something genreric like 'filter' rather than truncating the two words somewhat arbitrarily.